atom/browser/api/atom_api_system_preferences_mac.mm

```diff
@@ -8,13 +8,57 @@
 
 #import <Cocoa/Cocoa.h>
 
+#include "atom/browser/mac/atom_application.h"
 #include "atom/browser/mac/dict_util.h"
 #include "atom/common/native_mate_converters/gurl_converter.h"
 #include "atom/common/native_mate_converters/value_converter.h"
 #include "base/strings/sys_string_conversions.h"
 #include "base/values.h"
+#include "native_mate/object_template_builder.h"
 #include "net/base/mac/url_conversions.h"
 
+namespace mate {
+template <>
+struct Converter<NSAppearance*> {
+  static bool FromV8(v8::Isolate* isolate,
+                     v8::Local<v8::Value> val,
+                     NSAppearance** out) {
+    if (val->IsNull()) {
+      *out = nil;
```

Comment:
If would be slightly clearer to `return true;` here and remove the `else` on the next line